ui/lib/core/addon/components/replication-header.js

```diff
@@ -4,5 +4,6 @@ import layout from '../templates/components/replication-header';
 export default Component.extend({
   layout,
   isSecondary: null,
+  secondaryId: null,
```

Comment:
could we check for `secondaryId` instead of using `isSecondary` ?  basically consolidating those attributes

Reply:
Good thought, but they're pretty different, both of these properties are passed down from different places.  So `secondaryId`comes from replicationAttrs model, and the `isSecondary` is a computed property set on the top parent of the contextual components that this component is nested under.  And in some cases you can have `isSecondary` = true, but null for `secondaryId` (that happens if you demote a DR Primary to Secondary.